libraries/AP_HAL_ChibiOS/hwdef/common/stm32h7_A3_mcuconf.h

```diff
@@ -0,0 +1,586 @@
+/*
+    ChibiOS - Copyright (C) 2006..2018 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+/*
+  this header is modelled on the one for the Nucleo-144 H743 board from ChibiOS
+ */
+#pragma once
+
+// we want to cope with both revision XY chips and newer chips
+#if !defined(HAL_CUSTOM_MCU_CLOCKRATE) || HAL_CUSTOM_MCU_CLOCKRATE <= 400000000
```

Comment:
Is this right? I thought it had a lower clock rate. Does it even have X&Y revs?

Reply:
cut&paste error, fixed